client/src/components/pages/search-page/search-task/search-task.module.scss

```diff
@@ -21,4 +9,12 @@
 .checkboxContainer {
 	display: flex;
 	flex-direction: column;
+	margin-bottom: 20px;
+}
+
+h6 {
+	margin: 20px 0px 10px 0px;
+	font-family: "Montserrat", sans-serif;
+	font-weight: 600;
+	font-size: 16px;
```

Comment:
better use variables from the file.
If there is no correct variable for this - create a new one or use old and add some needed styles